src/web/handlers/PanGestureHandler.ts

```diff
@@ -351,6 +361,70 @@ export default class PanGestureHandler extends GestureHandler {
     }
   }
 
+  private scheduleWheelEnd(event: AdaptedEvent) {
+    this.endWheelTimeout = setTimeout(() => {
+      if (this.receivedWheelEvent) {
+        return;
+      }
+
+      this.end();
+      this.tracker.removeFromTracker(event.pointerId);
+      this.currentState = State.UNDETERMINED;
+
+      this.wheelDevice = WheelDevice.UNDETERMINED;
+    }, 30);
```

Comment:
Shouldn't this be configurable?

Reply:
I'm not sure, should it? I was to move it to a constant but it seems that I've missed it. Either way, do we want to give users permission to change this value? I think we can stick to one that works ok, but leaving it open for customization also makes sense.

Reply:
I guess we can leave it for now and see if there's a demand for it.